CHANGELOG.md

```diff
@@ -3,7 +3,48 @@
 
 ## Version 0.42.0
 
-In Development.
+([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.41.0...v0.42.0))
+
+### Highlights
+
+🚀 Many thanks to **first time contributors** @gluonhiggs, @JMuff22 and @Shivansh20128!
+
+💡 A new error-mitigation technique is on its way. Thanks to @Misty-W for the Probabilistic Error Amplification RFC!
```

Comment:
Can you add a link to the RFC? Let's say that it has been reviewed and accepted, and we will begin implementing the technique in Q1.